modules/collect/src/main/java/com/opengamma/strata/collect/function/IntLongConsumer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2019 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.collect.function;
+
+/**
+ * An operation consuming two arguments - {@code int} and {@code long}.
+ * <p>
+ * Implementations of this interface will operate using side-effects.
+ */
+@FunctionalInterface
+public interface IntLongConsumer {
```

Comment:
Worth checking if these functional interfaces are the only ones you are going to need for `long`.